packages/@coorpacademy-components/src/template/app-review/slides/index.js

```diff
@@ -0,0 +1,354 @@
+import React, {useState, useMemo, useEffect} from 'react';
+import classnames from 'classnames';
+import get from 'lodash/fp/get';
+import getOr from 'lodash/fp/getOr';
+import isNil from 'lodash/fp/isNil';
+import {ICON_VALUES} from '../../../atom/review-header-step-item';
+import ReviewBackground from '../../../atom/review-background';
+import ReviewCongrats from '../../../organism/review-congrats';
+import ReviewHeader from '../../../organism/review-header';
+import ReviewCorrectionPopin from '../../../molecule/review-correction-popin';
+import Answer from '../../../molecule/answer';
+import ButtonLink from '../../../atom/button-link';
+import style from './style.css';
+import propTypes from './prop-types';
+
+const stylesByPosition = {
+  0: style.position0,
+  1: style.position1,
+  2: style.position2,
+  3: style.position3,
+  4: style.position4
+};
+
+export const TOTAL_SLIDES_STACK = 5;
+export const HIGHEST_INDEX = TOTAL_SLIDES_STACK - 1;
+
+const getSlideAnimation = (action, position, hidden) => {
+  switch (action) {
+    case 'unstack':
+      return style.slideOutHideAndIn;
+    case 'restack':
+      return style.slideOutAndIn;
+    default:
+      return hidden ? style.hiddenSlide : stylesByPosition[position];
+  }
+};
+
+const buildSlide = (
```

Comment:
ne faut il pas plutot transformer ça en component ?

```
const Slide = props => ...
```

pour que ce soit bien optimisé par le cycle de vie de React

Reply:
j'ai eu le meme point sur app-podcast; 
doc ici https://dev.to/igor_bykov/react-calling-functional-components-as-functions-1d3l

tu en penses quoi @StefanoA1 ?

Reply:
"React will compare the output of the function, but not the function itself"

Reply:
@chrisdugne c'est cool l'info du lien, je vais migrer ça, j'avais hesité avant mais oui t'as raison